src/main/java/com/exasol/releaserobot/usecases/validate/BasicMavenPomValidator.java

```diff
@@ -0,0 +1,65 @@
+package com.exasol.releaserobot.usecases.validate;
+
+import java.util.logging.Logger;
+
+import com.exasol.releaserobot.repository.GitRepository;
+import com.exasol.releaserobot.repository.maven.JavaMavenGitBranchContent;
+import com.exasol.releaserobot.repository.maven.MavenPom;
+import com.exasol.releaserobot.usecases.*;
+
+/**
+ * This class validates a pom file.
+ */
+public class BasicMavenPomValidator implements RepositoryValidator {
```

Comment:
It looks like this is not a BasicMavenPomValidator, but rather a broader concept like a MavenRepositoryValidator, which checks that a Maven repository has everything in place.

Maybe we should change the name, and also move it to its right package (not in the usecases package)

Reply:
Renamed and moved